### PR TITLE
Swedish Translation

### PR DESCRIPTION
**Pull request by Lazylllama**
Here is my Swedish translation using a bit of Swedish slang words and also normal Swedish. If there's any swedes who wants to correct me leave a comment please.

### CODE REVIEW

**Comment by JayXTQ** [comment]
Hi, please read translations/README.md and look at the other translations if you can. You have some mistakes inside of your translation (i.e. flags aren’t supposed to be there). Thanks!

**Comment by Lazylllama** [comment]
My apologies. Did this at like 1 in the mornin 😅
Should be all good now.

File: translations/README_sv_SE.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+- [Spectre](https://spectreattack.com/spectre.pdf)
+- [Meltdown](https://meltdownattack.com/meltdown.pdf)
+- [DMA Remapping](https://docs.microsoft.com/en-us/windows/security/information-protection/kernel-dma-protection-for-thunderbolt)
+- [(P) ATMFD Exploit](https://msrc.microsoft.com/update-guide/en-US/vulnerability/CVE-2020-1020)
+- [(P) Print Nightmare](https://us-cert.cisa.gov/ncas/current-activity/2021/06/30/printnightmare-critical-windows-print-spooler-vulnerability)
+- [Remote Desktop](https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=Windows+Remote+Desktop)
+- [NetBIOS](https://en.wikipedia.org/wiki/NetBIOS) (_Possible Information Retrieval_)
```
Comment (PencilNavigator) [inline translations/README_sv_SE.md:50]: (_Possible Information Retrieval_) should be translated

File: README.md [hunk 2]
```diff
@@ -31,6 +31,7 @@
 <kbd>[<img title="Türkçe" alt="Türkçe" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/tr.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_tr_TR.md)</kbd>
 <kbd>[<img title="Українська" alt="Українська" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/ua.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_ua_UA.md)</kbd>
 <kbd>[<img title="Tiếng Việt" alt="Tiếng Việt" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/vn.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_vi_VN.md)</kbd>
+<kbd>[<img title="Svenska" alt="Svenska" src="https://cdn.staticaly.com/gh/hjnilsson/country-flags/master/svg/se.svg" width="22">](https://github.com/Atlas-OS/Atlas/blob/main/translations/README_sv_SE.md)</kbd>
```
Comment (PencilNavigator) [inline README.md:34]: put it after line 30 (Tagalog) since it should be in alphabetical order.

**Review by PencilNavigator — CHANGES_REQUESTED** [review]
so far so good, just need a little change.
Also, add the language chooser in ./transalations/README.md

**Comment by JayXTQ** [comment]
Looks good, just someone will need to translate as I can not right now. If nobody comes back to this I will check in a bit

**Comment by Lazylllama** [comment]
> so far so good, just need a little change. Also, add the language chooser in ./transalations/README.md
*Both are solved.*

**Review by PencilNavigator — APPROVED** [review]
Format is good.

**Pull request merged**